diff --git a/node_modules/@distube/yt-dlp/dist/index.js b/node_modules/@distube/yt-dlp/dist/index.js
index b17b126..3fd7f30 100644
--- a/node_modules/@distube/yt-dlp/dist/index.js
+++ b/node_modules/@distube/yt-dlp/dist/index.js
@@ -71,18 +71,22 @@ Status code ${response.statusCode.toString()}`);
 }, "makeRequest");
 var args = /* @__PURE__ */ __name((url, flags = {}) => [url].concat((0, import_dargs.default)(flags, { useEquals: false })).filter(Boolean), "args");
 var json = /* @__PURE__ */ __name((url, flags, options) => {
+  if (typeof url === "string" && url.startsWith("-")) {
+    return Promise.reject(new Error(`Invalid URL: ${url}`));
+  }
   const process2 = (0, import_node_child_process.spawn)(YTDLP_PATH, args(url, flags), options);
   return new Promise((resolve, reject) => {
     let output = "";
+    let errorOutput = "";
     process2.stdout?.on("data", (chunk) => {
       output += chunk;
     });
     process2.stderr?.on("data", (chunk) => {
-      output += chunk;
+      errorOutput += chunk;
     });
     process2.on("close", (code) => {
       if (code === 0) resolve(JSON.parse(output));
-      else reject(new Error(output));
+      else reject(new Error(errorOutput || output));
     });
     process2.on("error", reject);
   });
